src/main/resources/hudson/plugins/junitattachments/AttachmentTestAction/sidepanel.jelly

```diff
@@ -1,3 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
That is not the problem. The problem is line 5, where we are still calling a now-deprecated method which is not Pipeline-compatible.